parm/FV3LAM_wflow.xml

```diff
@@ -1188,6 +1189,46 @@ the <task> tag to be identical to the ones above for other output times.
   {%- endif %}
 {%- endif %}
 {%- endif %}
+{%- if run_task_run_prdgen %}                      
+<!--
+************************************************************************
+************************************************************************
+-->
+<metatask name="&TN_RUN_PRDGEN;{{ uscore_ensmem_name }}">
+
+    <var name="fhr">{% for h in range(0, fcst_len_hrs+1) %}{{ " %03d" % h }}{% endfor %}
+</var>
+    <task name="&TN_RUN_PRDGEN;{{ uscore_ensmem_name }}_f#fhr#" cycledefs="forecast" max
+tries="{{ maxtries_run_prdgen }}">                 
```

Comment:
There is a mistake here.

Reply:
Oh yeah the formatting got messed up somehow here.  I'll fix that